File: Conferencia/Services/Servers/Admin_Service/src/Config/OracleConnection.js
const oracledb = require('oracledb')

credentials = {
    user: process.env.Oracle_User,
    password: process.env.Oracle_Pass,
    connectString: process.env.Oracle_Dsn
}

try {
    oracledb.initOracleClient({libDir: '/opt/oracle/instantclient_19_3'})
    console.log("Oracle Connected!!!")
}catch (err) {
    console.error('No Se Establecio La Conexion')
}

async function Open(query, binds, autoCommit){
    let connection = await oracledb.getConnection(credentials);
    let result = await connection.execute(query, binds, {autoCommit})
    connection.release();
    return result;
}

exports.Open = Open;

File: Conferencia/Services/Servers/Admission_Service/src/services/oracle_connection.py
import cx_Oracle
from flask import json
from flask.json import jsonify
import os

class Connection:             
        connection = None        

        def __init__ (self):
                cx_Oracle.init_oracle_client(lib_dir=r"/opt/oracle/instantclient_19_3")
                self.connection = cx_Oracle.connect(os.environ['oracle_user'], os.environ['oracle_pass'], os.environ['oracle_dsn'])
                print('Conexion Iniciada!!!')      

        def login(self, user, password):
                cursor = self.connection.cursor()
                cursor.execute("""
                        SELECT LOGIN(:0, :1)
                        FROM dual""", [user, password])
                                
                for fname in cursor:
                   return fname

        def signUp(self, user, password):
                try:
                        cursor = self.connection.cursor()
                        cursor.callproc('INSERT_USER', [user, password, 'U'])
                        return jsonify({"auth": True})
                except cx_Oracle.Error as e:
                        print(e)
                        return jsonify({"auth": False})


        def getConnection(self):                
                return self.connection